app/uk/gov/hmrc/hec/models/CTStatus.scala

```diff
@@ -21,32 +21,19 @@ import ai.x.play.json.SingletonEncoder.simpleName
 import ai.x.play.json.implicits.formatSingleton
 import play.api.libs.json.Format
 
-sealed trait CTStatus extends Product with Serializable {
-  val IFString: String
-}
+sealed trait CTStatus extends Product with Serializable
 
 object CTStatus {
-  case object ReturnFound extends CTStatus {
-    override val IFString: String = "Return Found"
-  }
-  case object NoticeToFileIssued extends CTStatus {
-    override val IFString: String = "Notice to File Issued"
-  }
-  case object NoAccountingPeriodFound extends CTStatus {
-    override val IFString: String = "No Accounting Period Found"
-  }
-  case object NoReturnFound extends CTStatus {
-    override val IFString: String = "No Return Found"
-  }
+
+  case object ReturnFound extends CTStatus
```

Comment:
the strings are now numbers, so I thought it'd be nicer to have those where we're using them in  `IFServiceImpl` rather than having magic numbers over here. What do you think?

Reply:
agree with you 👍